gems/sorbet/test/hidden-method-finder/BUILD

```diff
@@ -35,10 +36,19 @@ hidden_methods_tests(
     ],
 )
 
+hidden_methods_tests(
+    ruby = "sorbet_ruby_3_0",
+    tests = [
+        "simple",
+        "thorough",
+    ],
+)
+
 test_suite(
     name = "hidden-method-finder",
     tests = [
         ":hidden-methods-sorbet_ruby_2_6",
```

Comment:
Ruby 2.6 goes EOL in 2 days. Do you want to just delete the tests now to save time?

https://www.ruby-lang.org/en/downloads/branches/

Reply:
We still depend on it for rubyfmt, so I'll just remove the testing that depends on it for now.